### PR TITLE
Fix #64 issue

### LINKED ISSUE #64: Import-D365AadUser / AadGroupID
Idea : Make a solution (new parameter ?) to search the exact AadGroup name and not a "like" operator to be sure that we can get the correct Group

for exemple, i have 2 group

Group_A
Group_A_B

i want to use the command for Group_A, Currently that not working because the commande say "More than 1 group found

or develop a capabilities to send the ObjectId for AadGroup (witch is unique)

### PR DESCRIPTION
**Pull request by Shind44**
Please check if our developpement standard was respected

Charles

### CODE REVIEW

File: functions/import-d365aaduser.ps1 [hunk 1]
```diff
@@ -157,15 +178,37 @@ function Import-D365AadUser {
...
+        }
 
         if ($null -eq $group) {
             Write-PSFMessage -Level Host -Message "Unable to find the specified group in the AAD. Please ensure the group exists and that you have enough permissions to access it."
             Stop-PSFFunction -Message "Stopping because of errors"
             return
         }
+        else
+        {
+            Write-PSFMessage -Level Host -Message "Processing Azure AD user Group ""$($group[0].DisplayName)"""
```
Comment (Splaxi) [inline functions/import-d365aaduser.ps1]: Write-PSFMessage -Level Host -Message "Processing Azure AD user Group $($group[0].DisplayName)"

Comment (Shind44) [reply functions/import-d365aaduser.ps1]: Mötz, it seems that working with my "" 
image


but i change this line to apply our recommdation and create a new pull request

Comment (Shind44) [reply functions/import-d365aaduser.ps1]: hum the code formatting is better with your solution, see directly that a variable is added to a string content

**Comment by Shind44** [comment]
Add #65 (Full Text Catalog problem) in this PR
> Update SQL file to add my SQL custom command

**Comment by Splaxi** [comment]
Did you run a test with the current code changes?

**Comment by Shind44** [comment]
Mötz : of course, i never commit or Pull code withtout make at leat some unit tests

**Comment by Splaxi** [comment]
Then I'll merge it and make it part of the release tonight.

**Comment by Shind44** [comment]
Yeah

**Pull request merged**